objects.py

```diff
@@ -14,31 +13,45 @@ def __init__():
 
         # This should set the object's price and calories, accessible by
         # .price and .calories respectively.
+        """ For creating desserts """
+        self.price = price
+        self.calories = calories
         pass
 
     # Add a calories_per_dollar method that returns the calories per dollar
     # for the dessert.
+    def calories_per_dollar(self):
+        if self.calories != None:
```

Comment:
A slightly more Pythony way would be `is not None` rather than `!= None`, though both work. :)